src/vector/index.html

```diff
@@ -50,7 +50,11 @@
         <% }
     } %>
   </head>
-  <body style="height: 100%; margin: 0;" data-vector-indexeddb-worker-script="<%= htmlWebpackPlugin.files.chunks['indexeddb-worker'].entry %>">
+  <body
+      style="height: 100%; margin: 0;"
+      data-vector-indexeddb-worker-script="<%= htmlWebpackPlugin.files.chunks['indexeddb-worker'].entry %>"
+      data-vector-recorder-worklet-script="<%= htmlWebpackPlugin.files.chunks['recorder-worklet'].entry %>"
```

Comment:
@gsouquet is upgrading this plugin in https://github.com/vector-im/element-web/pull/17039, so you may need to resolve a conflict here.

Reply:
conflicts indeed, but thankfully straightforward. That change also looks neat :D